### PR TITLE
fix(ONYX-1097): adjust component structure

### PR DESCRIPTION
**Pull request by dariakoko**
The type of this PR is: **Fix**

This PR solves [ONYX-1097]

### Description

Adjust the Notification component structure to support displaying the component from the top and not from the previous scroll position

Before: https://artsy.slack.com/archives/C03N12SR0RK/p1719489084109179

After:

https://github.com/artsy/force/assets/36167539/a3a44651-a637-43e6-bd2c-793afa5256cd

[GRO-434]: https://artsyproduct.atlassian.net/browse/GRO-434?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ
[ONYX-1097]: https://artsyproduct.atlassian.net/browse/ONYX-1097?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

File: src/Components/Notifications/Notification.tsx [hunk 1]
```diff
@@ -104,36 +105,50 @@ const Notification: React.FC<NotificationProps> = ({ notificationId }) => {
...
-      )
-    case "VIEWING_ROOM_PUBLISHED":
-      return (
-        <ViewingRoomPublishedNotification
-          notification={data.me?.notification}
-        />
-      )
-    default:
-      return null
+  const renderNotificationObject = data => {
```
Comment (dzucconi) [inline src/Components/Notifications/Notification.tsx:108]: The linter should be preventing this?

**Comment by dzucconi** [comment]
Why not just scroll to top on route change?

File: src/Components/Notifications/Notification.tsx (hunk 1 above)
Comment (dzucconi) [reply src/Components/Notifications/Notification.tsx:108]: https://github.com/artsy/force/blob/af8878df0bc133ef75b15b591c0df2917d27a6b9/.eslintrc.js#L70

https://github.com/jsx-eslint/eslint-plugin-react/blob/master/docs/rules/no-unstable-nested-components.md

**Review by nickskalkin — COMMENTED** [review]
Looks good to me! Maybe fix what Damon said about declaring nested components here: https://github.com/artsy/force/pull/14133/files#r1662371659

**Review by nickskalkin — APPROVED** [review]
👏

**Comment by dariakoko** [comment]
Will go ahead and merge, thanks all for your comments!

**Pull request merged**